<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,80,1,0">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="Log">
            <attribute name="title">
             <string>Log</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QTextBrowser" name="textLog"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Terminal">
            <attribute name="title">
             <string>Terminal</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QTextBrowser" name="textTerminal"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Curve">
            <attribute name="title">
             <string>Curve</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,30,0">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <property name="spacing">
              <number>1</number>
             </property>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1" rowspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="12,1">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2" stretch="15,1">
                 <property name="spacing">
                  <number>1</number>
                 </property>
                 <item>
                  <widget class="QChartView" name="widget" native="true"/>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_2" native="true">
                   <widget class="QSlider" name="horizontalSlider">
                    <property name="geometry">
                     <rect>
                      <x>10</x>
                      <y>10</y>
                      <width>951</width>
                      <height>22</height>
                     </rect>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <widget class="QCheckBox" name="cbxCH1">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>51</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>CH1</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="labCH1">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>40</y>
                    <width>61</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                 <widget class="QCheckBox" name="cbxCH2">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>80</y>
                    <width>51</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>CH2</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="labCH2">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>110</y>
                    <width>61</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Plot</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_plot" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,40,1,1,0">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QTextEdit" name="textSend"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnSend">
           <property name="font">
            <font>
             <family>Adobe Devanagari</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <underline>true</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnClean">
           <property name="font">
            <font>
             <family>Adobe Devanagari</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <underline>true</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>清除</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionlianjie"/>
    <addaction name="actionduankai"/>
    <addaction name="actiontuichu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actionqingchu"/>
    <addaction name="actionshowLog"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlianjie">
   <property name="icon">
    <iconset>
     <normaloff>../../../../.designer/backup/ico/开关-关.png</normaloff>../../../../.designer/backup/ico/开关-关.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="actionduankai">
   <property name="icon">
    <iconset>
     <normaloff>../../../../.designer/backup/ico/开关-开.png</normaloff>../../../../.designer/backup/ico/开关-开.png</iconset>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </action>
  <action name="actiontuichu">
   <property name="icon">
    <iconset>
     <normaloff>../../../../.designer/backup/ico/退出.png</normaloff>../../../../.designer/backup/ico/退出.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionqingchu">
   <property name="text">
    <string>清除窗口</string>
   </property>
  </action>
  <action name="actionshowLog">
   <property name="text">
    <string>显示Log</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>PyQt5/QtChart</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
